Description: add libwlroots-0.18-dev headers include path
Author: Safir Secerovic <sapphire@linux.org.ba>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/meson.build
+++ b/src/meson.build
@@ -25,7 +25,7 @@
 avif_dep = dependency('libavif', version: '>=1.0.0', required: get_option('avif_screenshots'))
 
 wlroots_dep = dependency(
-  'wlroots',
+  'wlroots-0.18',
   version: ['>= 0.18.0'],
   fallback: ['wlroots', 'wlroots'],
   default_options: ['default_library=static', 'examples=false', 'xwayland=enabled', 'backends=libinput', 'renderers=[]', 'allocators=[]', 'session=enabled'],
@@ -87,7 +87,8 @@
 reshade_include = include_directories([
   'reshade/source',
   'reshade/include',
-  '/usr/include/spirv/unified1'
+  '/usr/include/spirv/unified1',
+  '/usr/include/wlroots-0.18'
 ])
 
 sol2_include = include_directories(['../thirdparty'])
